Content.Server/Clothing/EntitySystems/ToggleableClothingSystem.cs

```diff
@@ -0,0 +1,174 @@
+using System.Threading.Tasks;
+using Content.Shared.Actions;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Clothing.EntitySystems;
+using Content.Shared.Toggleable;
+using Content.Shared.Inventory;
+using Content.Server.Body.Components;
+using Content.Server.Clothing.Components;
+using Content.Server.Disease.Components;
+using Content.Server.Nutrition.EntitySystems;
+using Content.Server.Atmos.Components;
+using Content.Server.Popups;
+using Robust.Shared.Player;
+
+namespace Content.Server.Clothing.EntitySystems
+{
+    public sealed class ToggleableClothingSystem : SharedToggleableClothingSystem
```

Comment:
why was all this moved to server. afaict almost all of this can still remain in shared